src/common/low_precision_transformations/src/reduce_sum.cpp

```diff
@@ -68,7 +68,10 @@ void ReduceSumTransformation::changeDequantizationValues(
         }
 
         // (a1 - s) + (a2 - s) + ... + (an - s) = (a1 + a2 + ... + an) - n * s
-        const auto reductionSizeConstant = ov::opset1::Constant::create(deqPrecision, Shape{}, { static_cast<float>(reductionSize) });
+        const auto reductionSizeConstant = ov::opset1::Constant::create(
+            dequantization.subtractConstant->get_element_type(),
```

Comment:
In case of low precision overflow is possible, I think we should avoid it. Let's discuss it offline

Reply:
thanks, fixed